.github/workflows/cicd-release-validation.yml

```diff
@@ -0,0 +1,580 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# This is the top-level workflow that runs on release branches.
+# It invokes other jobs to perform builds and run tests.
+# All jobs run in parallel, using build artifacts to synchronize jobs.
+#
+# If you add or remove any tests that use reusable-test.yml on a pull request,
+# you must update codecov.yml to match.
+
+name: CI/CD - Release validation
+
+on:
+  # Permit manual runs of the workflow.
+  workflow_dispatch:
+    inputs:
+      # Request a target release branch to run CI/CD on.
+      branch:
```

Comment:
Can we make this trigger off the creation of the release tag?

Reply:
Yes, and also added automatic triggering on release branches.